src/dist/replication/lib/replica_bulk_load.cpp

```diff
@@ -71,7 +73,115 @@ void replica::on_bulk_load(const bulk_load_request &request, /*out*/ bulk_load_r
 // ThreadPool: THREAD_POOL_REPLICATION
 void replica::broadcast_group_bulk_load(const bulk_load_request &meta_req)
 {
+    if (!_primary_states.learners.empty()) {
+        dwarn_replica("has learners, skip broadcast group bulk load request");
+        return;
+    }
+
+    if (!_primary_states.group_bulk_load_pending_replies.empty()) {
+        dwarn_replica("{} group bulk_load replies are still pending, cancel it firstly",
+                      static_cast<int>(_primary_states.group_bulk_load_pending_replies.size()));
```

Comment:
no need to static_cast
```suggestion
                      _primary_states.group_bulk_load_pending_replies.size());
```